Update lead paragraph of creating-custom-views-for-blocklist.md (#6382)
umbraco/UmbracoDocs: 14/umbraco-cms/tutorials/creating-custom-views-for-blocklist.md

# Custom Views for Block List

> Custom Views are used to overwrite the presentation of a Block. We can improve the editing experience by providing a better representation of the Blocks content. This can be as it will look on the front-end or specific properties of its content that gets presented.

Reviewer: ```suggestion
Custom Views are used to overwrite the presentation of a Block. We can improve the editing experience by providing a better representation of the Blocks content. It could be a presentation of how it will look on the front end or the specific properties available.
```
